configs/phi-1_5/setup/requirements.txt

```diff
@@ -12,9 +12,11 @@ sentencepiece==0.2.0
 trl==0.8.6
 protobuf==3.20.3
 ipykernel==6.29.4
-onnxruntime-gpu==1.17.1
+onnxruntime-gpu==1.19.0
 einops==0.8.0
 promptflow==1.10.1
 promptflow-tools==1.4.0
 gradio==4.29.0
-olive-ai==0.6.0
\ No newline at end of file
+onnxruntime-genai-cuda==0.4.0
+optimum==1.21.4
+git+https://github.com/microsoft/Olive@753583ab0f04be962de65cf15acb8b3fbca3c511
```

Comment:
The ACA image lacks Git, preventing the new changes from functioning in the remote environment.

Reply:
Updated PR to add a .whl file which will be used instead of git. This should work for the remote scenario